loopr-ios/Setting/SettingManageWallet/SettingWalletDetail/ChangeWalletName/SettingChangeWalletNameViewController.swift

```diff
@@ -27,40 +31,41 @@ class SettingChangeWalletNameViewController: UIViewController, UITextFieldDelega
         let padding: CGFloat = 15
         
         // Do any additional setup after loading the view.
-        self.navigationItem.title = LocalizedString("Change Wallet Name", comment: "")
+        self.navigationItem.title = LocalizedString("Wallet Name", comment: "")
         setBackButton()
         
         nameTextField.delegate = self
         nameTextField.tag = 0
+        nameTextField.theme_textColor = GlobalPicker.textColor
         nameTextField.font = FontConfigManager.shared.getDigitalFont()
-        nameTextField.theme_tintColor = GlobalPicker.textColor
+        nameTextField.backgroundColor = UIColor.dark2
         nameTextField.placeholder = "Enter your wallet name"
         nameTextField.contentMode = UIViewContentMode.bottom
-        nameTextField.frame = CGRect(x: padding, y: originY, width: screenWidth-padding*2, height: 40)
+        nameTextField.placeHolderColor = UIColor.dark4
+        nameTextField.borderColor = UIColor.dark3
+        nameTextField.borderWidth = 1
+        nameTextField.setLeftPaddingPoints(padding)
+        nameTextField.frame = CGRect(x: 0 - nameTextField.borderWidth, y: originY, width: screenWidth + nameTextField.borderWidth * 2, height: 51)
```

Comment:
You should use padding. It's fine to use xib if you want. line 48 is not correct.